doctr/models/recognition/master/pytorch.py

```diff
@@ -206,17 +213,19 @@ def decode(self, encoded: torch.Tensor) -> torch.Tensor:
         b = encoded.size(0)
 
         # Padding symbol + SOS at the beginning
-        ys = torch.full((b, self.max_length), self.vocab_size + 2, dtype=torch.long, device=encoded.device)
-        ys[:, 0] = self.vocab_size + 1
+        ys = torch.zeros((b, self.max_length), dtype=torch.long, device=encoded.device) * (self.vocab_size + 2)  # pad
```

Comment:
?
previously, the tensor was full of the value `self.vocab_size + 2`
now, it's full of zeros (multiplying 0 by `self.vocab_size + 2` will yield zero :sweat_smile:

Reply:
Oh f...k got it :sweat_smile: 
In the end, it's really just about having the shape, it doesn't matter what values ​​are in there. 
I also assumed that it would only make sense to fill with the PAD value, but that's irrelevant while inference.
But to make it make some sense, I'll change it :+1: